web/src/features/panels/zone/FeedbackCard.cy.tsx

```diff
@@ -1,13 +1,20 @@
+import FeedbackCard from 'components/app-survey/FeedbackCard';
```

Comment:
Show we have a component test like his for the new SurveyCard as well? :)

Reply:
It's pretty much the same card but with different text, I can make one but it didn't feel super useful when I looked at it